.circleci/local_publish_helpers_codebuild.sh

```diff
@@ -214,12 +213,11 @@ function retry {
     n=0
     FAILED_TEST_REGEX_FILE="./amplify-e2e-reports/amplify-e2e-failed-test.txt"
     rm -f $FAILED_TEST_REGEX_FILE
-    until [ $n -ge $MAX_ATTEMPTS ]
-    do
+    until [ $n -ge $MAX_ATTEMPTS ]; do
         echo "Attempting $@ with max retries $MAX_ATTEMPTS"
         setAwsAccountCredentials
         "$@" && break
-        n=$[$n+1]
+        n=$(($n + 1))
```

Comment:
are you using some custom .sh formatter ?

Reply:
reverted.